### PR TITLE
Documentation(Ej2-846715): modified the RowTemplate

### PR DESCRIPTION
**Pull request by Gayathri4135**
modified the row template general content, row template with formatting content.

### CODE REVIEW

File: ej2-javascript/code-snippet/grid/row-template-component-cs1/index.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { Grid } from '@syncfusion/ej2-grids';
+import { data } from './datasource.ts';
+import { ChipList } from '@syncfusion/ej2-buttons';
+import { NumericTextBox } from '@syncfusion/ej2-inputs';
+import { DatePicker } from '@syncfusion/ej2-calendars';
+import { DropDownList } from '@syncfusion/ej2-dropdowns';
+
+let grid: Grid = new Grid({
+    dataSource: data,
+    rowTemplate: '#rowtemplate',
```
Comment (BalajiS4008) [inline ej2-javascript/code-snippet/grid/row-template-component-cs1/index.ts:10]: ej2-javascript/code-snippet/grid/row-template-component-cs1/index.js - You have defined height and width but here those properties are missed, please ensure that

File: ej2-javascript/code-snippet/grid/row-template-component-cs1/datasource.ts
Comment (BalajiS4008) [inline ej2-javascript/code-snippet/grid/row-template-component-cs1/datasource.ts:1]: The technical codes and content are fine and need to check demo of these topices

File: ej2-javascript/code-snippet/grid/row-template-component-cs1/index.ts (hunk 1 above)
Comment (Gayathri4135) [reply ej2-javascript/code-snippet/grid/row-template-component-cs1/index.ts:10]: In row-template-component-cs1/index.js - height and width properties are unnecessary so I removed it

File: ej2-javascript/code-snippet/grid/row-template-component-cs1/js/index.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>EJ2 Grid</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="Typescript Grid Control">
+    <meta name="author" content="Syncfusion">
+    <link href="index.css" rel="stylesheet">
```
Comment (BalajiSGrid) [inline ej2-javascript/code-snippet/grid/row-template-component-cs1/js/index.html:10]: referred themes are very old version, please check and change require files

File: ej2-javascript/grid/row/row-template.md [hunk 3]
```diff
@@ -73,6 +81,43 @@ If the [`rowTemplate`](../../api/grid/#rowtemplate) is used, the value cannot be
...
+{% include code-snippet/grid/row-template-component-cs1/index.html %}
+{% endhighlight %}
+{% endtabs %}
+
+{% previewsample "page.domainurl/code-snippet/grid/row-template-component-cs1" %}
+{% endif %}
+
 ## Limitations
 
 Row template feature is not compatible with all the features which are available in grid and it has limited features support. Here we have listed out the features which are not compatible with row template feature.
```
Comment (BalajiSGrid) [inline ej2-javascript/grid/row/row-template.md:123]: changes is not apply on limitation topic

File: ej2-javascript/code-snippet/grid/row-template-component-cs1/js/index.html (hunk 2 above)
Comment (Gayathri4135) [reply ej2-javascript/code-snippet/grid/row-template-component-cs1/js/index.html:10]: changed version

File: ej2-javascript/grid/row/row-template.md (hunk 3 above)
Comment (Gayathri4135) [reply ej2-javascript/grid/row/row-template.md]: changed the content of limitation

**Comment by Gayathri4135** [comment]
Changes moved into new branch #130  . So i closed this PR

**Pull request closed without merging**